googleapis/google-cloud-node
# migrate code from googleapis/nodejs-filestore

- feat: initial stub of library
- feat!: initial generation of filestore API (#2)
- chore: release 1.0.0 (#3)
- chore(deps): update dependency sinon to v11 (#5)
- chore(deps): update dependency mocha to v9 (#4)
- docs(filestore): Fix various formatting issues and broken links (#6)
- chore(nodejs): update client ref docs link in metadata (#9)
- fix(build): migrate to using main branch (#11)
- fix(deps): google-gax v2.24.1 (#13)
- chore: release 1.0.1 (#12)
- feat: turns on self-signed JWT feature flag (#14)
- chore: release 1.1.0 (#15)
- docs(samples): add auto-generated samples for Node with api short name in region tag (#21)
- fix(filestore): add missing operation_metadata.proto import (#24)
- build(node): update deps used during postprocessing (#1243) (#26)
- chore: release 1.1.1 (#25)
- chore: fix the wrong post processor image (#28)
- chore(deps): update dependency @types/node to v16 (#27)
- chore(deps): update dependency sinon to v12 (#29)
- docs(samples): add example tags to generated samples (#30)
- build: add generated samples to .eslintignore (#31)
- docs(node): support "stable"/"preview" release level (#1312) (#34)
- chore: add api_shortname and library_type to repo metadata (#33)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#36)
- test(nodejs): remove 15 add 16 (#1322) (#38)
- build: update copyright year to 2022 (#41)
- chore: update v2.12.0 gapic-generator-typescript (#44)
- chore(deps): update dependency sinon to v13 (#46)
- docs(nodejs): version support policy edits (#1346) (#49)
- chore(main): release 1.1.2 (#47)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#50)
- chore: update v2.14.2 gapic-generator-typescript (#54)
- build(node): update client library version in samples metadata (#1356) (#61)
- chore(deps): update dependency sinon to v14 (#64)
- build!: update library to use Node 12 (#66)
- chore(main): release 2.0.0 (#67)
- chore(deps): update dependency jsdoc-fresh to v2 (#69)
- chore(deps): update dependency jsdoc-region-tag to v2 (#70)
- feat: support regapic LRO (#71)
- chore(main): release 2.1.0 (#72)
- chore(deps): update dependency linkinator to v4 (#76)
- fix: better support for fallback mode (#77)
- fix: change import long to require (#78)
- fix: do not import the whole google-gax from proto JS (#1553) (#81)
- fix: allow passing gax instance to client constructor (#82)
- fix: preserve default values in x-goog-request-params header (#83)
- test: use fully qualified request type name in tests (#85)
- fix(deps): use google-gax v3.5.2 (#89)
- chore(deps): update dependency jsdoc to v4 (#91)
- chore(deps): update dependency @types/node to v18 (#87)
- fix: regenerated protos JS and TS definitions (#92)
- feat: new APIs added to reflect updates to the filestore service (#93)
- Update .repo-metadata.json
- chore(main): release 2.2.0 (#79)
- Update .repo-metadata.json
- Update .repo-metadata.json
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
